### DISCUSSION TITLE
will adobe animate support Android App Bundle? Play Store Uploads warnings?

### DISCUSSION THREAD

**Discussion by ageekhere** [airsdk/Adobe-Runtime-Support#596] [Q&A]
Hi will adobe animate support Android App Bundle so a 3rd party tool is not needed? 

Also when using Android Applications – Play Store Uploads
1) Create a first APK file for 32-bit ARM (armv7)
2) Update the version number by as small as increment as possible
3) Create a second APK file for 64-bit ARM (armv8)
4) Upload both APK files to the Play Store.

This creates warnings for the armv7 APK  "this release is not compliant with google player 64-bit requirement" is this an issue?

**Comment by nboy1** [comment]
if you still can't figure out how to export aab i put a udemy course step by step about it i hope it can help
[here](https://www.udemy.com/course/flashtoandroidappbundle/?referralCode=CFBC85E2C62CBF80815D)

**Comment by marchbold** [comment]
Here's some references from other developers to guide you through the new AAB process: 

- https://medium.com/@kanellopoulos.leo/air-apache-flex-2021-how-to-compile-android-app-bundle-aacc72caa3db
- https://www.reddit.com/r/as3/comments/owag5h/building_aab_with_flash_develop_working_experience/
